src/app/pages/StakingPage/Features/ValidatorMediaInfo/index.tsx

```diff
@@ -17,8 +18,12 @@ export const ValidatorMediaInfo = memo((props: Props) => {
   return (
     <>
       {info.email_address && <MediaButton href={`mailto:${info.email_address}`} icon={<MailOption />} />}
```

Comment:
Is the email address validated anywhere in the stack? Otherwise some mail clients are vulnerable to autoattaching files with `mailto:?attach=..`

(https://www.nds.ruhr-uni-bochum.de/media/nds/veroeffentlichungen/2020/08/15/mailto-paper.pdf)